msg-bot.service

```diff
@@ -0,0 +1,20 @@
+# To use this file: symlink into /etc/systemd/system
+# Then enable with:
+# > systemctl enable msg-bot.service
+# > systemctl start msg-bot.service
+
+[Unit]
+Description=Message Bot Service
+After=network.target 
+StartLimitIntervalSec=60
+StartLimitBurst=5
+
+[Service]
+Type=simple
+WorkingDirectory=/home/zach/msg-bot

```

Comment:
Should eventually properly install this elsewhere & set a different directory for data and logs